generator.py

```diff
@@ -5,56 +5,97 @@
 random.seed()
 
 
-def generate(note_threads_number, accord_threads_number, length, tonality, positivity):
+def generate(note_threads_number, accord_threads_number, length, tonality, positivity, tempo):
     """
     this function produces melody which contains set of melodies: common, and accord-melodies
     :param note_threads_number: (int)how many flows of common melodies do we want
     :param accord_threads_number: (int)how many flows of accord-melodies do we want
     :param length: (int)how many notes should be in the melody
     :param tonality: (list) it list of allowed sounds
     :param positivity: (int) 0 or 1 0-major, 1-minor
+    :param tempo: (int) speed of melody, more tempo -> more speed
     :return: (list of Melody) all the melodies which music contains
     """
-    seq_seq = list()
+    melody_sequence = list()
+    patternise_number = 3
+    main_melody = create_melody_core(tonality, positivity, length // 2 ** (patternise_number - tempo - 1) + 1, 2 ** (patternise_number - tempo))
```

Comment:
make this multiline + use explicit parameter names.
e.g.

`create_melody_core(tonality, positivity, 
         length=length // 2 ** (patternise_number - tempo - 1) + 1, 
         duration=2 ** (patternise_number - tempo))`

Reply:
fixed